cmd/agent/app/processors/thrift_processor_test.go

```diff
@@ -219,8 +219,8 @@ func assertProcessorCorrectness(
 
 	// agentReporter must emit metrics
 	mTestutils.AssertCounterMetrics(t, metricsFactory, []mTestutils.ExpectedMetric{
-		{Name: "tchannel-reporter.batches.submitted", Tags: map[string]string{"format": format}, Value: 1},
-		{Name: "tchannel-reporter.spans.submitted", Tags: map[string]string{"format": format}, Value: 1},
+		{Name: "reporter.batches.submitted", Tags: map[string]string{"format": format}, Value: 1},
+		{Name: "reporter.spans.submitted", Tags: map[string]string{"format": format}, Value: 1},
```

Comment:
should this also have the `protocol` tag?

Reply:
They have the tag but it's set on the factory which is passed to `NewMetrics`... Before I had a parameter in the builder but removed per your and @objectiser request.